src/assisted_test_infra/test_infra/utils/utils.py

```diff
@@ -508,11 +508,18 @@ def get_openshift_release_image(allow_default=True):
         with open(consts.RELEASE_IMAGES_PATH, "r") as f:
             release_images = json.load(f)
 
-        release_image = [
-            v.get("url")
-            for v in release_images
-            if v.get("openshift_version") == ocp_version and v.get("cpu_architecture") == "x86_64"
-        ][0]
+        try:
+            release_image = [
+                v.get("url")
+                for v in release_images
+                if v.get("openshift_version") == ocp_version and v.get("cpu_architecture") == "x86_64"
```

Comment:
```suggestion
                if v.get("openshift_version") == ocp_version and (v.get("cpu_architecture") == "x86_64" or "x86_64" in v.get("cpu_architectures"))
```
This would make try..catch unnecessary

Reply:
Sure, fixed